frontend/src/pages/projects/screens/spawner/imageSelector/ImageSelectorField.tsx

```diff
@@ -52,21 +59,23 @@ const ImageSelectorField: React.FC<ImageSelectorFieldProps> = ({
     });
   };
 
-  if (error) {
+  if (error || currentProjectImageStreamsError) {
     return (
       <Alert title="Image loading error" variant="danger">
-        {error.message}
+        {error?.message || currentProjectImageStreamsError?.message}
```

Comment:
ide not smart enough to realize the second  `?.` isn't necessary because of the way the if block is set up? 😆 

could maybe add `const errorMessage = error?.message || currentProjectImageStreamsError?.message` above the `if` and then just have `if(errorMessage) ... {errorMessage}`

but no biggie.

Reply:
Done